appserver/packager/microprofile-package/pom.xml

```diff
@@ -224,7 +224,7 @@
             <artifactId>jersey-proxy-client</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.glassfish.jersey.ext</groupId>
```

Comment:
```
[ERROR]   The project org.glassfish.main.packager:microprofile-package:5.192-SNAPSHOT (/home/mark/git/payara/appserver/packager/microprofile-package/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.glassfish.jersey.ext.microprofile:jersey-mp-rest-client:jar is missing. @ org.glassfish.main.packager:microprofile-package:[unknown-version], /home/mark/git/payara/appserver/packager/microprofile-package/pom.xml, line 226, column 21
```
without looking, due to dependency managment needing updating too perhaps?

Reply:
BOM changed in payara/Payara_PatchedProjects#248. After clearing local repo for `org.glassfish.jersey:jersey-bom` it should build fine.